doc/massdeploymentcommandline.rst

```diff
@@ -0,0 +1,30 @@
+It is possible to perform mass deployment of the Nextcloud desktop client by passing certain command-line parameters from the deployment step to Nextcloud desktop client executable after the initial setup.
+This will allow desktop client to generate a config (.cfg) file that will be used during subsequent launches.
+A config file will have a corresponding account written into it similar to if you have added it manually via the desktop client's UI.
+The desktop client will exit with code 0 if account has been added successfuly, or 1 in case of failure.
+Detailed failure message is printed to the desktop client logs.
+
+The following parameters are supported:
+
+``--userid``
+        (required) userId (username as on the server) to pass when creating an account via command-line.
+
+``--apppassword``
+        (required) appPassword to pass when creating an account via command-line (see the ``login-flow`` section in server documentation on how to generate the app password).
+
+``--localdirpath``
+        (optional) path where to create a local sync folder when creating an account via command-line. If skipped, then default local sync folder path (/home/Nextcloud<n>) will be generated by desktop client.
```

Comment:
`/home/Nextcloud<n>` is not very descriptive
on Windows that would be something else and on Linux that would usually be `/home/<userid>/Nextcloud`